.travis.yml

```diff
@@ -1,14 +1,15 @@
 language: rust
 sudo: false
 rust:
```

Comment:
Can we continue testing against our minimum supported Rust version, which'll be 1.4.0 I believe.